packages/lib-sourcify/src/lib/types.ts

```diff
@@ -30,13 +30,15 @@ export interface MissingSources {
   };
 }
 
-export interface MetadataSources {
-  [index: string]: {
-    keccak256: string;
-    content?: string;
-    urls?: string[];
-    license?: string;
-  };
+export interface MetadataSource {
+  keccak256: string;
+  content?: string;
+  urls?: string[];
+  license?: string;
+}
+
+export interface MetadataSourceMap {
```

Comment:
This name was a bit confusing, but I see the fault is of `MetadataSource`, I honestly don't know a better name. Just leaving this comment to share this thought, but it's not blocking the PR.

Reply:
yeah yeah it's clear, I just didn't like the name. This is not a big deal, just do as you prefer.